src/utils.jl

```diff
@@ -221,6 +221,25 @@ For example `select_from_tuple(("a", "b", "c"), Val((3, 1, 1)))` returns
     return Expr(:tuple, [Expr(:ref, :t, k) for k in P]...)
 end
 
+@doc raw"""
+    symmetrize!(Y,X)
+
+Given a quare matrix `X` compute `1/2 .* (X' + X)` in place of `Y`
+"""
+function symmetrize!(Y, X)
+    Y .= 1 / 2 .* (X' .+ X)
```

Comment:
```suggestion
    Y .= (X' .+ X) ./ 2
```
This avoids using `Float64`.

Reply:
I would (again) not have thought of that.